reference/5-api-publisher/Use-TextFormatter-Serilog.md

```diff
@@ -0,0 +1,106 @@
+# How to use the TextFormatter class
+
+API Publisher uses by default the following output template in the _logging.json_ configuration:
+```
+[{Timestamp:yyyy-MM-dd HH:mm:ss,fff}] [{Level:u4}] [{ThreadId:00}] {SourceContext} - {Message} {Exception} {NewLine}
+```
+
+This template uses the format provided by the **Serilog** library, you can find more details on how to create your specific format:
+[Formatting Output](https://github.com/serilog/serilog/wiki/Formatting-Output)
+
+The sinks available in API Publisher are Console, File, and Aws CloudWatch. Some of these sinks have compatibility with the default output template, but in some cases that doesn't work, for example the Aws CloudWatch sink. If you run into this problem, we provide our TextFormatter which is similar to the default. Here we are going to see how to use it:
+
+## How to use it in the AWSCloudWatch:
+
+In the _logging.json_ file
+
+- Using the default format
+```
+"WriteTo:AWSCloudWatch": {
+  "Name": "AmazonCloudWatch",
+  "Args": {
+    "logGroup": "Ed-Fi-Publisher",
+    "logStreamPrefix": "Ed-Fi-Tools",
+    "restrictedToMinimumLevel": "Verbose",
+    "textFormatter": "EdFi.Tools.ApiPublisher.Core.Configuration.Serilog.TextFormatter, EdFi.Tools.ApiPublisher.Core"
+  }
+}
+```
+
+> [!NOTE]
+> The default template format provided by the TextFormatter is: ```[{Timestamp:yyyy-MM-dd HH:mm:ss,fff}] [{Level}] [{ThreadId:00}] [{SourceContext}] - {Message} {Exception} {NewLine}```

```

Comment:
Another opportunity for a Docusaurus-style admonition.

Reply:
fixed